packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -976,7 +976,12 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
       useMaterial3: useMaterial3,
     );
 
-    final TextStyle? effectiveTextStyle = widget.textStyle ?? theme.textStyle ?? defaults.textStyle;
+    final TextStyle? baseTextStyle = widget.textStyle ?? theme.textStyle ?? defaults.textStyle;
+    final Color disabledColor = Theme.of(context).colorScheme.onSurface.withOpacity(0.38);
```

Comment:
With this change, seems we are not able to customize disabled color. I think we can add a `bool enabled` property in `_DropdownMenuDefaultsM3` and provide this value as the default value when `DropdownMenu` is disabled. Does this make sense?